docs/trace/building-your-own-sampler/MySampler.cs

```diff
@@ -20,6 +20,6 @@ internal class MySampler : Sampler
 {
     public override SamplingResult ShouldSample(in SamplingParameters samplingParameters)
     {
-        return new SamplingResult(false);
+        return new SamplingResult(Decision.NotRecord);

```

Comment:
I wonder if we could make `SamplingResult(bool)` overload to make it easier for people who struggle with tri-state.